install/kubernetes/experimental-install.yaml

```diff
@@ -325,15 +327,8 @@ rules:
   resources:
   - nodes
   verbs:
-  # To annotate node
+  # To annotate the k8s node with Cilium's metadata
   - update
```

Comment:
Should this be update nodes/status?

Reply:
^ I was reviewing patch-by-patch here, I think that this _should_ have been addressed by the last patch, but if so, github should mark this comment as out-of-date because it gets updated. Do we need to re-generate the experimental / quick install YAMLs? IIRC v1.9 was the last release where we distributed these dedicated YAMLs.